src/main/resources/assets/bloodmagic/lang/es_ES.lang

```diff
@@ -0,0 +1,164 @@
+#Creative Tab
+itemGroup.BloodMagic.creativeTab=Magia Sanguinea
+itemGroup.BloodMagic.creativeTabTome=Tomos de Mejora de Magia Sanguinea
+
+#Items
+item.BloodMagic.activationCrystal.weak.name=Cristal de Activación Débil
+item.BloodMagic.activationCrystal.awakened.name=Cristal de Activación Despertado
+item.BloodMagic.activationCrystal.creative.name=Cristal de Activación Creativo
+
+item.BloodMagic.sacrificialDagger.normal.name=Daga Sacrifical
+item.BloodMagic.sacrificialDagger.creative.name=Daga Sacrifical Creativa
+item.BloodMagic.pack.sacrifice.name=Escudo de Armas
+item.BloodMagic.daggerOfSacrifice.name=Daga de Sacrificio
+
+item.BloodMagic.lavaCrystal.name=Cristal de Lava
+
+item.BloodMagic.bound.sword.name=Espada Confinada
+item.BloodMagic.bound.pickaxe.name=Pico Confinado
+item.BloodMagic.bound.axe.name=Hacha Confinado
+item.BloodMagic.bound.shovel.name=Pala Confinada
+
+item.BloodMagic.bucket.lifeEssence.name=Cubo de Vida
+
+item.BloodMagic.scribe.water.name=Herramienta de Inscripción Elemental: Agua
+item.BloodMagic.scribe.fire.name=Herramienta de Inscripción Elemental: Fuego
+item.BloodMagic.scribe.earth.name=Herramienta de Inscripción Elemental: Tierra
+item.BloodMagic.scribe.air.name=Herramienta de Inscripción Elemental: Aire
+item.BloodMagic.scribe.dusk.name=Herramienta de Inscripción Elemental: Crepúsculo
+item.BloodMagic.scribe.dawn.name=Herramienta de Inscripción Elemental: Alba
+
+item.BloodMagic.focus.weak.name=Foco de Teleposición
+item.BloodMagic.focus.enhanced.name=Foco de Teleposición Mejorado
+item.BloodMagic.focus.reinforced.name=Foco de Teleposición Reforzado
+item.BloodMagic.focus.demonic.name=Foco de Teleposición Demoníaco
+
+item.BloodMagic.slate.blank.name=Loja en Blanco
+item.BloodMagic.slate.reinforced.name=Loja Reforzada
+item.BloodMagic.slate.imbued.name=Loja Imbuída
+item.BloodMagic.slate.demonic.name=Loja Demoníaca
+item.BloodMagic.slate.ethereal.name=Loja Etérea
+
+item.BloodMagic.orb.weak.name=Orbe de Sangre Débil
+item.BloodMagic.orb.apprentice.name=Orbe de Sangre de Aprendiz
+item.BloodMagic.orb.magician.name=Orbe de Sangre de Mago
+item.BloodMagic.orb.master.name=Orbe de Sangre de Maestro
+item.BloodMagic.orb.archmage.name=Orbe de Sangre de Archimago
+item.BloodMagic.orb.transcendent.name=Orbe de Sangre Transcendente
+
+item.BloodMagic.reagent.incendium.name=Incendium
```

Comment:
I didn't touch any but aether, since that one has a proper translation in spanish (éter). Makes sense though, I'll remove the stuff that is not changed that I left in the pull request by mistake.